content/reference/changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2017-07-09
+  * [doc update] Update `los_records` endpoint `id` has been migrated to BigInteger.
```

Comment:
This is not a doc update but an api one.
This can be a breaking change as well for some partners so need to be clearly stating it.